src/main/native/pairhmm/IntelPairHmm.cc

```diff
@@ -35,8 +35,14 @@ JNIEXPORT void JNICALL Java_com_intel_gkl_pairhmm_IntelPairHmm_initNative
 {
   DBG("Enter");
 
-  JavaData javaData;
-  javaData.init(env, readDataHolder, haplotypeDataHolder);
+  JavaData javaData(env);
+  try {
+    javaData.init(readDataHolder, haplotypeDataHolder);
+  } catch (JavaException& e) {
+    env->ExceptionClear();
```

Comment:
Add `javaData.releaseData(env);` before return

Reply:
Not necessary as `releaseData` gets called inside the destructor.